metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/UpdateIndicesHook.java

```diff
@@ -180,15 +183,45 @@ private void updateFineGrainedEdgesAndRelationships(
     }
   }
 
+  private Urn generateSchemaFieldUrn(@Nonnull String resourceUrn, @Nonnull String fieldPath) {
+    // we rely on schemaField fieldPaths to be encoded since we do that with fineGrainedLineage on the ingestion side
+    String encodedFieldPath = fieldPath.replaceAll("\\(", "%28").replaceAll("\\)", "%29").replaceAll(",", "%2C");
```

Comment:
nit: always good to make any fields that aren't going to change final

Reply:
Also for any function parameters!

Reply:
sounds good!